src/course_bot/quiz.clj

```diff
@@ -134,11 +135,12 @@
                  (talk/change-branch tx :approve {:quiz-key quiz-key})))
              :approve
              (fn [tx {{id :id} :from text :text} {quiz-key :quiz-key}]
-               (case (str/lower-case text)
-                 "yes" (do
-                         (talk/send-text token id (tr :quiz/quiz-started))
-                         (start-quiz! tx quiz-key))
-                 "no" (do (talk/send-text token id (tr :quiz/quiz-canceled)) (talk/stop-talk tx))))))
+               (let [normalized-text (i18n/normalize-yes-no-text text)]
+                 (case normalized-text
+                   "yes" (do
+                           (talk/send-text token id (tr :quiz/quiz-started))
+                           (start-quiz! tx quiz-key))
+                   (talk/process-answer token id tx normalized-text (tr :quiz/quiz-canceled) (tr :talk/question-yes-no)))))))
```

Comment:
Let's inline `talk/process-answer`. I think it will looks simpler here and in other places. And where it is applicable, use `send-stop` and `clarify-input` where it is applicable.

```suggestion
                 (case normalized-text
                   "yes" (do (talk/send-text token id (tr :quiz/quiz-started))
                             (start-quiz! tx quiz-key))
                   "no" (talk/send-stop tx token id)
                   (talk/clarify-input tx token id
                                            (format (tr :talk/clarify-input-1) text))))))

;; in talk
(defn send-stop
  ([tx token id] (send-cancel-talk token id (tr :talk/stop-talk)))
  ([tx token id msg] (send-text token id msg)
                     (stop-talk tx)))

;; and similar to clarify question
```